app/pandas.ipynb

```diff
@@ -369,9 +353,7 @@
    "cell_type": "code",
    "execution_count": null,
    "outputs": [],
-   "source": [
-    "# use * operator here"
-   ],
+   "source": "multiplication = s1 *",
```

Comment:
The multiplication operation on the Series 's1' is incomplete. You need to specify the second operand for the multiplication.